app/services/activity/updater.rb

```diff
@@ -36,7 +36,6 @@ def set_identifier
 
     def set_roda_identifier
       assign_attributes_for_step("roda_identifier_fragment")
```

Comment:
Does this step still make any sense?

Reply:
At this point in the feature, yeah, but it does need removing in a lter commit